gitbutler-ui/src/lib/components/SegmentControl/Segment.svelte

```diff
@@ -37,7 +37,7 @@
 	role="tab"
 	aria-selected={isSelected}
 	aria-disabled={disabled}
-	tabindex={isSelected ? 0 : -1}
+	tabindex={!isSelected ? 0 : -1}
```

Comment:
@PavelLaptev Consider avoiding this double-negative.

```suggestion
	tabindex={isSelected ? -1 : 0}
```

Reply:
Thanks!